channels/br.m3u

```diff
@@ -9,12 +21,50 @@ https://5a7bcfb834235.streamlock.net/fiocruz/fiocruz/playlist.m3u8?DVR
 http://tvajuhls-lh.akamaihd.net/i/tvdesk_1@147040/master.m3u8
 #EXTINF:-1 tvg-id="Canção Nova" tvg-name="Canção Nova" tvg-language="Portuguese" tvg-logo="https://i.imgur.com/zJSUbTM.png" group-title="Religious",Canção Nova
 https://cancaonova.secure.footprint.net/egress/bhandler/cancaonovanew/channel01/chunklist_b1155072.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Portuguese" tvg-logo="" group-title="",CBN São Paulo
+https://medias.sgr.globo.com/hls/vCBNSP/vCBNSP.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Portuguese" tvg-logo="" group-title="",CBN Rio
+https://medias.sgr.globo.com/hls/vCBNRJ/vCBNRJ.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Portuguese" tvg-logo="http://i.imgur.com/SYPYeHt.jpg" group-title="",Cine+
 http://189.86.89.116/hls-live/livecm/_definst_/liveevent/livestream.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Portuguese" tvg-logo="" group-title="",COM Brasil
+#EXTINF:-1 tvg-id="cnt.br" tvg-name="CNN" tvg-language="Portuguese" tvg-logo="" group-title="",CNN Brasil
+https://streaming.cnnbrasil.com.br:443/cnndigital_mainOutput2.m3u8
+#EXTINF:-1 tvg-id="cnt.br" tvg-name="CNT" tvg-language="Portuguese" tvg-logo="https://i.imgur.com/GVu2RbW.png" group-title="",CNT
+https://cdn0.showcase.com.br/hls/cnt/cwb.m3u8
```

Comment:
Links does not work.
Tested in Elmedia Video Player, VLC Player and streamtest.in

Reply:
Seems that these links are geo-restricted to Brazil.